<launch>
    <arg name="CAM_ID" default="0"/>
    <arg name="CAM_RESOLUTION" default=""/>
    <arg name="ENTER_MIN" default="1.0"/>
    <arg name="EXIT_MAX" default="0.5"/>
    <arg name="SHOW_CAM" default="true"/>
    <arg name="REQUIRED" default="true"/>
    <arg name="NAME" default="apriltag_detector"/>

    <node name="$(arg NAME)"
        pkg="apriltag_detector"
        type="main.py"
        output="screen"
        required="$(arg REQUIRED)">
        <param name="CAM_ID" value="$(arg CAM_ID)"/>
        <param name="CAM_RESOLUTION" value="$(arg CAM_RESOLUTION)"/>
        <param name="ENTER_MIN" value="$(arg ENTER_MIN)"/>
        <param name="EXIT_MAX" value="$(arg EXIT_MAX)"/>
        <param name="SHOW_CAM" value="$(arg SHOW_CAM)"/>
    </node>
</launch>